### PR TITLE
Update icons in Modelica.Mechanics.MultiBody.Visualizers from Wolfram MathCore

### PR DESCRIPTION
**Pull request by beutlich**
Use nicer icons from Wolfram SystemModeler for

- [x] Modelica.Mechanics.MultiBody.Visualizers
- [x] Modelica.Mechanics.MultiBody.Visualizers.Ground
- [x] Modelica.Mechanics.MultiBody.Visualizers.Torus
- [x] Modelica.Mechanics.MultiBody.Visualizers.VoluminousWheel
- [x] Modelica.Mechanics.MultiBody.Visualizers.PipeWithScalarField
- [x] Modelica.Mechanics.MultiBody.Visualizers.Advanced.PipeWithScalarField

@maltelenz @henrikt-ma For MSL v3.2 the icons were designed by @d-hedberg at Wolfram MathCore. Is it okay for you to take these six new icons for regular MSL again?

### CODE REVIEW

**Comment by tobolar** [comment]
What is the actual reason for this changes? To prefer image definition inside of a class instead of by an external image?

**Comment by beutlich** [comment]
No, to have nicer icons.

File: Modelica/Mechanics/MultiBody/Visualizers.mo [hunk 1]
```diff
@@ -829,36 +829,7 @@ with respect to frame_a (vector from the origin of frame_a to the arrow tail).
...
-            Text(
-            extent={{-150,-105},{150,-145}},
-            textString="%name",
-            textColor={0,0,255})}), Documentation(info="<html>
+       Icon(coordinateSystem(preserveAspectRatio = true, extent = {{-100, -100}, {100, 100}}, initialScale = 0.1, grid = {10, 10}), graphics = {Polygon(lineColor = {255, 255, 255}, fillColor = {126, 181, 78}, fillPattern = FillPattern.Solid, points = {{-100, -30}, {20, -90}, {100, 0}, {-10, 40}}), Text(textColor = {64, 64, 64}, extent = {{20, 70}, {60, 100}}, textString = "z", horizontalAlignment = TextAlignment.Left), Polygon(lineColor = {255, 255, 255}, fillColor = {14, 111, 1}, fillPattern = FillPattern.Solid, points = {{100, -10}, {20, -100}, {20, -90}, {100, 0}}), Polygon(lineColor = {255, 255, 255}, fillColor = {14, 111, 1}, fillPattern = FillPattern.Solid, points = {{-100, -40}, {20, -100}, {20, -90}, {-100, -30}}), Line(origin = {6, -8}, points={{-6,-10},{-6,108}}), Polygon(origin = {6, 0}, points={{-6,102},{-14,72},{2,72},{-6,102}}, fillPattern=FillPattern.Solid), Text(textColor = {64, 64, 64}, extent = {{-150, -145}, {150, -105}}, textString = "%name")}), Documentation(info = "<html>
```
Comment (tobolar) [inline Modelica/Mechanics/MultiBody/Visualizers.mo]: Any reason to set initialScale and grid?

**Comment by tobolar** [comment]
Considering rendering in Dymola's package browser, `Joints` icon seems not to be better/nicer to me. Other components are fine.

beutlich_better-icons_browser2

Dymola's diagram:

beutlich_better-icons

**Comment by maltelenz** [comment]
The rendering of the Joints icon looks fine in SystemModeler, and reasonable in OpenModelica, so I think this is a Dymola problem.
SystemModeler:
Screenshot_20190829_172152
OpenModelica:
Screenshot_20190829_172837

**Comment by beutlich** [comment]
Here's how the Joints icon looks in SimulationX.

MSL 3.2.3:
grafik

This PR:
grafik

I also prefer that we should stick with the old icon since drawing cylindric gradients seems problematic in small icons.

**Comment by beutlich** [comment]
And here's how the other new icons look in SimulationX:

grafik

**Comment by tobolar** [comment]
OK, let's focus on the Joints icon only. I'm going to prepare yet another atempt this morning and will let you know.

**Comment by tobolar** [comment]
Here are my attempts to improve the icons. In most cases, Dymola renders the cylinders poorly. In `Joints6` I've used the same principle like in Torus and VoluminousWheel, i.e. overlayed polygons.  This seems fine to me. 

beutlich_better-icons_Joints_browser2

@maltelenz, @beutlich: Please check in other tools whether this could be a solution.
https://gist.github.com/tobolar/95e62615d30ed5931cfd94c245040756

**Comment by maltelenz** [comment]
I added a 7 that is the 6 with a border.
I also fixed a typo in "rotation" in 1.
https://gist.github.com/maltelenz/098b8acaa0f079ff93aec93dcfd1187e

All look fine in SystemModeler:
Screenshot_20190830_121241

This is OpenModelica:
Screenshot_20190830_121158

To be honest, Dymolas rendering just looks broken, especially at the small sizes.

**Comment by beutlich** [comment]
Screen shot from SimulationX:

grafik

**Comment by tobolar** [comment]
> Broken tools should be ignored when writing a standard library for a language.

In that case I would vote for Joints4 or Joints5. What I in particular preferre is: darker colors, larger circles representing the flange/bolt

**Comment by maltelenz** [comment]
I like Joints4 as well.

**Comment by beutlich** [comment]
Already did so, This PR is on the Visualizers only. If you like you can approve,

File: Modelica/Mechanics/MultiBody/Visualizers.mo (hunk 1 above)
Comment (tobolar) [reply Modelica/Mechanics/MultiBody/Visualizers.mo]: @beutlich Sorry for bother you. Could you check my question?

Comment (beutlich) [reply Modelica/Mechanics/MultiBody/Visualizers.mo]: This is for @maltelenz .

Comment (maltelenz) [reply Modelica/Mechanics/MultiBody/Visualizers.mo]: No, I don't see a reason to set initialScale or grid.

Comment (beutlich) [reply Modelica/Mechanics/MultiBody/Visualizers.mo]: Can you please create a new PR to remove it. Thanks.

Comment (tobolar) [reply Modelica/Mechanics/MultiBody/Visualizers.mo]: Will do.

Comment (tobolar) [reply Modelica/Mechanics/MultiBody/Visualizers.mo]: See PR #3105

**Pull request merged**